dataenv.js.org

pgamerx opened the pull request:
Demo: https://pgamerx.github.io/dataenv/

I will add the CNAME once this is approved!

- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But you have to (re-)add a file named ```CNAME``` to your branch with the single line ```dataenv.js.org``` to make it work.
I'll merge after we have "contact". Please keep this PR open.

pgamerx commented:
Done, I have added the CNAME file!

On Thu, 3 Feb 2022 at 21:39 Stefan Keim ***@***.***> wrote:

> I've already added your requested subdomain to JS.ORGs zonefile. But you
> have to (re-)add a file named CNAME to your branch with the single line
> dataenv.js.org to make it work.
> I'll merge after we have "contact". Please keep this PR open.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/js-org/js.org/pull/6868#issuecomment-1029148018>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARCSGRYPJ7AKSVJPXHXPYYTUZKSDVANCNFSM5NN4PUZQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
*Regards,*
*       Naman Sharma (Website Designer)*

indus commented:
Welcome https://dataenv.js.org
